specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/Global.json

```diff
@@ -0,0 +1,110 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2016-03-01",
+    "title": "Global API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Web/billingMeters": {
```

Comment:
This should be referenced in the `readme.md` config file [here](https://github.com/andreyse/azure-rest-api-specs/blob/346651741dcafd1a91d0741d726aef7cdac29c4d/specification/web/resource-manager/readme.md)

Reply:
Thanks for review. I added it there